src/main/java/io/github/mountainrange/mule/MULE.java

```diff
@@ -116,6 +117,7 @@ public void setFullscreen(Boolean b) {
 	}
 
 	public void close() {
+		Config.getInstance().soundManager.stopAllPlaylists();
```

Comment:
Can we change this to 
```
Platform.exit();
System.exit(0);
```
if possible, if that dosent work, go ahead and merge this.